.circleci/config.yml

```
@@ -21,14 +21,29 @@ jobs:
       - run:
           name: Install Dependencies
           command: yarn
+      - run:
+          name: Build all code to JavaScript
+          command: yarn build
+      - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      - run:
+          name: Create CLI binaries
+          command: npx pkg --out-path ./cli-binaries ./packages/cli/
       - save_cache:
           name: Save Yarn Package Cache
           key: yarn-packages-{{ .Branch }}-{{ checksum "yarn.lock" }}
           paths:
             - node_modules/
+      - run:
+          command: SPEC=petstore.oas2.json BINARY=prism-cli-linux yarn run.binary
```

To make the harness cross platform, you can avoid to use the binary and use the source code directly:

```suggestion
          command: yarn workspace @stoplight/prism-cli cli mock ./examples/petstore.oas2.json
```

---

Doubling down on binaries, lets pass the entire path in instead of a partial, so the script is more flexible.

```suggestion
          command: SPEC=petstore.oas2.json BINARY=./cli-binaries/prism-cli-linux yarn run.binary
```